### PR TITLE
[WIP] Add bindings to gdf_nvtx_range_push and gdf_nvtx_range_pop.

### PR DESCRIPTION
**Pull request by shwina**
Bindings for functions in https://github.com/gpuopenanalytics/libgdf/pull/157

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by kkraus14** [comment]
add to whitelist

**Comment by nsakharnykh** [comment]
@jrhemstad can you add your changes to pygdf join and groupbys? and then we can merge I think

**Comment by scopatz** [comment]
LGTM!

File: pygdf/_gdf.py [hunk 1]
```diff
@@ -402,3 +402,48 @@ def count_nonzero_mask(mask, size):
...
+    """
+    return {
+        'green':    libgdf.GDF_GREEN,
+        'blue':     libgdf.GDF_BLUE,
+        'yellow':   libgdf.GDF_YELLOW,
+        'purple':   libgdf.GDF_PURPLE,
+        'cyan':     libgdf.GDF_CYAN,
+        'red':      libgdf.GDF_RED,
+        'white':    libgdf.GDF_WHITE,
+    }[s.lower()]
```
Comment (scopatz) [inline pygdf/_gdf.py]: Actually, it would be best if this dict was defined once globally, instead of each time the function was called.

**Comment by shwina** [comment]
@scopatz It turns out that `libgdf.RED` (for example) is a `int` in Python-land. So I had to fix some of the logic in `nvtx_range_push`. Please let me know if it still looks OK to you.

I also moved the dict to module scope.

**Comment by scopatz** [comment]
LGTM! Thanks @shwina

**Comment by dantegd** [comment]
rerun tests

**Comment by dantegd** [comment]
merging, thanks @shwina

**Pull request merged**